src/modules/sway/workspaces.cpp

```diff
@@ -322,7 +330,9 @@ bool Workspaces::handleScroll(GdkEventScroll *e) {
     }
   }
   try {
-    ipc_.sendCmd(IPC_COMMAND, fmt::format(workspace_switch_cmd_, name));
+    ipc_.sendCmd(
+        IPC_COMMAND,
+        fmt::format(workspace_switch_cmd_, "--no-auto-back-and-forth", name));
```

Comment:
Missing `config_["disable-auto-back-and-forth"].asBool()` no?

Reply:
Nope, disabling it when scrolling through workspaces on the bar is a behavior we want.
Technically, you can't "re-click" on the same workspaces when scrolling through them, but I chose to keep this behavior introduced by #736 since it seemed to fix some glitch.